bigbluebutton-html5/imports/api/2.0/voice-users/server/handlers/joinVoiceUser.js

```diff
@@ -4,6 +4,7 @@ import addVoiceUser from '../modifiers/addVoiceUser';
 
 export default function handleJoinVoiceUser({ body }, meetingId) {
   const voiceUser = body;
+  voiceUser.joined = true;
```

Comment:
I can't add it inside the modifiers, because the flow contract need for the voiceUsers.

The voiceUser is only add when `UserJoinedVoiceConfToClientEvtMsg` is handle, but, we need before, since we need the icons inside the user list, so we add an dummy voiceUser inside the `addUser`. If we add `joined` inside the `modifiers` we have the wrong state for the client, any better idea to solve this?